utils/index.js

```diff
@@ -131,6 +131,20 @@ const doRedirect = (req, res) => {
   return res.redirect(path)
 }
 
+const securityQuestionRedirect = (req, res, next) => {
+  const securityQuestionValues = req.session.login.securityQuestion
+
+  if (securityQuestionValues.length < 2) {
+    //TODO: redirect to the page they came from
+    return res.redirect('/login/securityQuestion')
+  } else if (securityQuestionValues.filter(v => v[0]).length < 2) {
+    //redirect to lite app
+    return res.redirect('/checkAnswers') 
+  }
+
+  next ()
```

Comment:
Is this valid? I would have thought that `next()` works, but `next ()` wouldn't.

Reply:
I should fix it anyways, because it is a bit strange